xds/internal/xdsclient/bootstrap/bootstrap.go

```diff
@@ -342,6 +342,8 @@ func NewConfigFromContents(data []byte) (*Config, error) {
 			if err := json.Unmarshal(v, &config.Authorities); err != nil {
 				return nil, fmt.Errorf("xds: json.Unmarshal(%v) for field %q failed during bootstrap: %v", string(v), k, err)
 			}
+		default:
+			logger.Infof("Bootstrap content has unknown field: %s", k)
```

Comment:
Should this be at warning level?

Reply:
Changed. I think it's lower than warning.